Src/Test.UnitTests.Sarif.PatternMatcher.Cli/ExportSearchDefinitionsCommandTests.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+
+using FluentAssertions;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
+{
+    public class ExportSearchDefinitionsCommandTests
+    {
+        [Fact]
+        public void ExportSearchDefinitionsCommand_ExportSingleBannedApi()
```

Comment:
I find following the Arrange, Action, Assert structure for tests to be really useful (http://wiki.c2.com/?ArrangeActAssert). It makes it really easy to read the tests, and maintain the tests going forward. Please treat this comment just as a suggestion.

Reply:
Added //Arrange //Act //Asssert comments